lib/hades/accounts/users.ex

```diff
@@ -34,5 +34,13 @@ defmodule Hades.Accounts.Users do
     end
   end
 
-  def delete_user(user), do: Repo.delete(user)
+  def update_user_status(user, attrs) do
+    changeset = User.changeset_update_user_status(user, attrs)
+    case Repo.update(changeset) do
```

Comment:
Is this `case` necessary? Maybe its a dumb question (I am new in Elixir), but since this `update` is returning exactly what we want to return, why check and return the exact same thing?

Reply:
I think this is for case when the user doesn't exist anymore (say, you deleted it in a different tab or something weird happened), in that case Ecto will return an error.

also if your changset is invalid (say bad password for example, or bad email format).

Reply:
You're right. Thanks